classes/log/EmailLogDAO.php

```diff
@@ -27,6 +27,8 @@
 
 class EmailLogDAO extends \PKP\db\DAO
 {
+    private int $MAX_SUBJECT_LENGTH = 255;
```

Comment:
Make this a `const` rather than a `private int`; also add a comment that this should match the `email_log` table's `subject` column length. Likewise, add a comment to `lib/pkp/classes/migration/install/LogMigration.php` to mention that the length should be kept the same as `EmailLogDAO::MAX_SUBJECT_LENGTH`.